macros/fivetran/base/fivetran_fb_ads.sql

```diff
@@ -0,0 +1,43 @@
+{% macro fivetran_fb_ads() %}
+
+    {{ adapter_macro('facebook_ads.fivetran_fb_ads') }}
+
+{% endmacro %}
+
+--there are multiple records for different statuses, but we don't need them for our purposes yet.
+--if this table needs to be expended to include multiple rows per id, this will break downstream queries that depend
+--on uniqueness on this id when joins are done.
+
+{% macro default__fivetran_fb_ads() %}
+
+with base as (
+
+    select distinct
```

Comment:
you don't want a distinct here if you're using the `row_number()` pattern to deduplicate! by definition you'll never have duplicates in this table because you have a row number on the row.